com.unity.render-pipelines.core/Runtime/Debugging/DebugFrameTiming.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+//#define RTPROFILER_DEBUG
+
+namespace UnityEngine.Rendering
+{
+    public class DebugFrameTiming
+    {
+        const string k_FpsFormatString = "{0:F1}";
+        const string k_MsFormatString = "{0:F2}ms";
+        const float k_RefreshRate = 1f / 5f;
```

Comment:
why this value for the refresh rate?

Reply:
It's the same value that [HDRP had](https://github.com/Unity-Technologies/Graphics/pull/5419/files#diff-9c446cfe3ccbc91b147ecf9663bcff3908953ec24bce72c3c7b3605ccc531b14L971) previously, so I didn't change it. I guess the refresh rate is a compromise between performance (it costs something to update the value), readability (too quickly changing values can be hard to read) and correctness (how up to date the value is). I think this value is a reasonable compromise.